ArduCopter/mode.h

```diff
@@ -1454,21 +1454,15 @@ class ModeAutorotate : public Mode {
     // --- Internal variables ---
     float _initial_rpm;             // Head speed recorded at initiation of flight mode (RPM)
     float _target_head_speed;       // The terget head main rotor head speed.  Normalised by main rotor set point
-    float _fwd_speed_target;        // Target forward speed (cm/s)
     float _desired_v_z;             // Desired vertical
     int32_t _pitch_target;          // Target pitch attitude to pass to attitude controller
-    float _collective_aggression;   // The 'aggresiveness' of collective appliction
-    float _z_touch_down_start;      // The height in cm that the touch down phase began
-    float _t_touch_down_initiate;   // The time in ms that the touch down phase began
     float now;                      // Current time in millis
```

Comment:
This is... weird.  Should be a local `const uint32_t` .  Pre-existing, of course.

Reply:
good catch, I've added a patch to fix it and related issues